Dockerfile

```diff
@@ -5,6 +5,7 @@ ENV GOVUK_APP_NAME rummager
 ENV REDIS_HOST redis
 ENV ELASTICSEARCH_URI http://elasticsearch:9200
 ENV PORT 3009
+ENV RABBITMQ_URL amqp://guest:guest@rabbitmq:5672
```

Comment:
Should this be a separate commit?

Reply:
It's only really relevant as part of the queue work as until now rummager didn't need to know about this ENV var until now

Reply:
Oh right I see as we use: https://github.com/alphagov/publishing-e2e-tests/blob/master/docker-compose.yml#L101-L104 as the way to connect.

It doesn't strike me as ideal to have different env vars that achieve the same task. 
Could we swap the Bunny.new to be either: `Bunny.new(GovukMessageQueueConsumer::RabbitMQConfig.from_environment(ENV))` or have the env vars inputted as a hash?

Ideally it'd be better still if GovukMessageQueueConsumer supported the RABBITMQ_URL env var, though we've had problems trying to make use of them since they support only a single host and we currently rely on a round robin collection of hosts (if I remember that stuff correctly)